karma/makeconf.js

```diff
@@ -72,15 +72,33 @@ function makeConf(defaultFile, browserNoActivityTimeout, requires) {
       browsers = ['ChromeWebRTC', 'FirefoxWebRTC'];
     }
 
+    const mochaOptions = {
+      require: requires,
+    };
+
+    // we'll mark our unstable tests with @unstable
+    switch  (process.env.TEST_STABILITY) {
+      case 'unstable':
+        mochaOptions.grep = '@unstable';
+        break;
+      case 'stable':
+        mochaOptions.grep = '@unstable';
+        mochaOptions.invert = 1;
+        break;
+      case 'all':
+        // when no grep specified, all tests will run.
+        break;
+      default:
+        console.log('invalid value for TEST_STABILITY:', process.env.TEST_STABILITY)
```

Comment:
is TEST_STABILITY mandatory? Instead of logging `invalid value`, we could log `Env not set` or something?

Reply:
`TEST_STABILITY` is not mandatory, it defaults to `all` - I will just remove this console.log to avoid the noise.